src/completion.ml

```diff
@@ -0,0 +1,292 @@
+(** Completion procedure for closed equations *)
+
+open Terms
+open Basics
+open Extra
+open Timed
+
+type 'a cmp = 'a -> 'a -> int
```

Comment:
For the sake of completeness, please add a comment on the semantics of the output (same as the one of OCaml std lib).